src/db/index.js

```diff
@@ -16,6 +16,7 @@ export default class {
 
     init(workbox) {
         registerRoutes(workbox, this);
+        this.openDB();
```

Comment:
Why do you do this if every call below also does it?  What's the value in opening it here?

Reply:
Only CRUD operations implicitly open the db before continuing. `openDb` initializes the db version, which is used in `/data/upload`. Consider the case when we run `/data/upload` first thing after we initialize the script. It is going to throw, since db version is not set.